packages/db/src/models/file.ts

```diff
@@ -1,9 +1,13 @@
 import Knex from 'knex';
+import * as s3Utils from '@cumulus/aws-client/S3';
+import { UnparsableFileLocationError, DeletePublishedGranule } from '@cumulus/errors';
```

Comment:
Definitely a nit but you can sort by alphabetical order here 🅰️ 🅱️ (there is no C emoji?? 😭 )

Reply:
Yup! Absolutely. This import is actually gone in the latest refactor but I can definitely do that going forward.